pkg/upstream/http.go

```diff
@@ -134,7 +135,7 @@ func newReverseProxy(target *url.URL, upstream options.Upstream, errorHandler Pr
 	}
 
 	// Apply the customized transport to our proxy before returning it
-	proxy.Transport = transport
+	proxy.Transport = otelhttp.NewTransport(transport)
```

Comment:
What happens if tracing isn't configured?

Reply:
I believe it will just pass through, like it was before.